charts/openshift-metering/templates/reporting-operator/reporting-operator-config.yaml

```diff
@@ -15,6 +15,9 @@ data:
   prometheus-url: {{ required "a valid reporting-operator.spec.config.prometheusURL must be set" $operatorValues.spec.config.prometheusURL | quote}}
   disable-prometheus-metrics-importer: {{ $operatorValues.spec.config.disablePrometheusMetricsImporter | quote}}
   enable-finalizers: {{ $operatorValues.spec.config.enableFinalizers | quote}}
+{{- if $operatorValues.spec.config.prestoTLS.enabled }}
+  presto-ca-file: "{{ $operatorValues.spec.config.prestoTLS.mountLocation }}/certificate.crt"
```

Comment:
I feel like we should use ca.crt or tls.crt here for consistency. How about we just use `tls.crt` for now. 

Also, I'm not entirely sure why we have `$operatorValues.spec.config.prestoTLS.mountLocation` at all? I don't think any of this needs to be configurable. Additionally, maybe don't use `/var/run/secrets/java.io/keystores`, that's a bit confusing since we're not using a java keystore file in this secret, just a CA certificate and not a java program. Maybe use `/var/run/secrets/presto-tls/tls.crt`